pydocx/export/base.py

```diff
@@ -779,7 +826,7 @@ def paragraph(self, text):
         return text
 
     @abstractmethod
-    def heading(self, text, heading_level):
+    def heading(self, text, heading_style_name):
```

Comment:
This makes this a backward-incompatible change, right? Shouldn't we document that change in the CHANGELOG and call it out, explicitly? Any customer parsers will need to update their definition.